src/pipeline.ml

```diff
@@ -509,47 +524,41 @@ module Ocaml_org = struct
     in
     pipelines, additional_build_args
 
-  let extras () =
-    let tarsnap =
-      let monthly = Current_cache.Schedule.v ~valid_for:(Duration.of_day 30) () in
-      Current_ssh.run ~schedule:monthly "watch.ocaml.org" ~key:"tarsnap" (Current.return ["./tarsnap-backup.sh"])
-    in
-    let peertube =
-      let weekly = Current_cache.Schedule.v ~valid_for:(Duration.of_day 7) () in
-      let image = Cluster.Watch_docker.pull ~schedule:weekly "chocobozzz/peertube:production-bookworm" in
-      Cluster.Watch_docker.service ~name:"infra_peertube" ~image ()
-    in
-    [tarsnap; peertube]
```

Comment:
Some of these changes relating to the `v` should have been in a separate commit, that was an oversight on my part. But they were made to simplify and support changes that were needed to complete the main change here, due to the altered representation of the service type.